scripts/raw_to_easymocap.py

```diff
@@ -0,0 +1,25 @@
+import os, sys, shutil
+
+DATA_TYPE = ['png','PNG','jpg','JPG']
+def create_input():
```

Comment:
Don't put `sys.argv` in function, it is great to use `__name__ == "__main__"` pattern.

We usually will add additional layer for this:

```python
def create_input(arg1, arg2, arg3...):
 ...

def main():
 # do what ever you want for `args`, e.g.
 filename = sys.argv[1]
 arg2 = sys.argv[2]
 ....

 # Then pass it into our function
 create_input(filename, arg2, arg3)

if __name__ == "__main__":
  main()
```

Reply:
Removed usage of `sys.argv` and instead made use of `python-click`